docs/integrations/hosts-operating-systems/opentelemetry/macos-opentelemetry.md

```diff
@@ -0,0 +1,163 @@
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-Schematics.png')} alt="Schematics" />
+
+## Fields Created in Sumo Logic for macOS
+
+Following are the tags which will be created as part of macOS App install, if not already present. 
+
+**Across apps**
+
+- **`sumo.datasource`** - Has fixed value of **mac**.
+
+## Collecting Logs, Metrics & installing macOS app
+
+Here are the steps for collecting metrics and installing the app.
+
+### Step 1: Set up Collector
+
+If you want to use an existing Otel Collector then this step can be skipped by selecting the option of using an existing Collector.
+
+If you want to create a new Collector please select **Add a new Collector** option.
+
+Select the platform for which you want to install the Sumo OpenTelemetry Collector.
+
+This will generate a command which can be executed in the machine which needs to get monitored. Once executed it will install the Sumo Logic OpenTelemetry Collector agent.  
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-Collector.png')} alt="Collector" />
+
+### Step 2: Configure integration
+
+In this step we will be configuring the yaml required for macOS Collection.
+
+There are no parameters required for the macOS app to install. One can add a custom tag if required in this step.
+
+Click on the **Download YAML File** button to get the yaml file.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Mac-OpenTelemetry/Mac-YAML.png')} alt="YAML" />
+
+:::note
+By default the collector will be sending process metrics to Sumo Logic. Since the number of processes running can be very large, this may result in significant increase in Data Points per Minute (DPM) If you would like to narrow down the list of processes being monitored, this can be done by adding the following entry under the process section of the downloaded yaml.
+```sh
+process:
+  include:
+    names: [ <process name1>, <process name2> ... ]
+    match_type: <strict|regexp>
+```
+:::
+
+### Step 3: Sending metrics to Sumo
+
+Once you have the yaml file downloaded in step 2, follow the below steps :
+
+1.  Copy the yaml file to `/etc/otelcol-sumo/conf.d/` folder in the macOS instance which needs to be monitored.
+2.  Restart the otelcol-sumo process using the below command 
+```sh
+ otelcol-sumo --config /etc/otelcol-sumo/sumologic.yaml --config "glob:/etc/otelcol-sumo/conf.d/*.yaml"
+```
+
+After successful execution of the above command, Sumo will start receiving the data from your host machine.
+
+Press **Next**. This will install the app in your Sumo Logic Org. The app consists of dashboards and monitors.
+
+Panels will start to fill automatically. It's important to note that each panel fills with data matching the time range query and received since the panel was created. Results won't immediately be available, but with a bit of time, you'll see full graphs and map.
```

Comment:
```suggestion
Panels will start to fill automatically. It's important to note that each panel fills with data matching the time range query and received since the panel was created. Results won't immediately be available, but within 20 minutes, you'll see full graphs and map.
```